### DISCUSSION TITLE
Dark mode and VMD - invisible text

### DISCUSSION THREAD

**Discussion by balazsfabian** [Colvars/colvars#665] [Ideas]
I used the Colvars Dashboard on my Mac, where I prefer the dark mode for all applications.
For some reason, VMD does not seem to fully support it. As a result, in the white textbox of the
Colvar config editor I get white text, which is of course invisible unless highlighted. A good
workaround is to exclude VMD from the dark mode applications. I post it here in the hope
that is saves some people the time:

```
# Check whether VMD is found by the 'defaults' command
defaults read | grep VMD

# Change the appearance for VMD
defaults write VMD NSRequiresAquaSystemAppearance -boolean true
```

**Comment by jhenin** [comment]
Thanks @balazsfabian ! I am converting this to a Discussion, where it can stay "open" for a long time, which will make it most useful.

**Comment by jhenin** [comment]
For context, VMD's UI is a mix of FLTK (linked to C++ code), and Tk or ttk, called from Tcl. The widgets from the Dashboard are mostly ttk, so that decides what is or isn't supported.
I see this kind of question has been [discussed before in the context of ttk](https://wiki.tcl-lang.org/page/Tk+differences+on+Mac+OS+X). This can be queried, so there is probably a way to write a workaround. That's a lot of platform-specific code though.

Anyway, thank you @balazsfabian  for providing your global workaround for VMD.
